modules/assim.sequential/R/sda.enkf_MultiSite.R

```diff
@@ -425,6 +425,13 @@ sda.enkf.multisite <- function(settings,
           }) %>%
           stats::setNames(site.ids)
         
+        #if it's a rabbitmq job sumbmission, we will first copy and paste the whole run folder within the SDA to the remote host.
+        if (!is.null(settings$host$rabbitmq)) {
+          cmd <- paste0("oc rsync", settings$host$rundir, "$(", "oc" )
```

Comment:
This line doesn't seem to get used anywhere

Reply:
Fixed.